browser/ui/tabs/split_view_tab_tile_data.cc

```diff
@@ -0,0 +1,257 @@
+/* Copyright (c) 2024 The Brave Authors. All rights reserved.
```

Comment:
this is presumably just a rename and the added comments kept git from recognizing that it's a move?

Reply:
Well.. git think it's not trivial filename change as it has class name/var name changes.